bin/config-optimist.js

```diff
@@ -1,48 +1,94 @@
 module.exports = function(optimist) {
 	optimist
-		.boolean("help").alias("help", "h").alias("help", "?").describe("help")
-		.string("config").describe("config", "Path to the config file")
-		.string("config-name").describe("config-name", "Name of the config to use")
-		.string("env").describe("env", "Environment passed to the config, when it is a function")
-		.string("context").describe("context", "The root directory for resolving entry point and stats")
-		.string("entry").describe("entry", "The entry point")
-		.string("module-bind").describe("module-bind", "Bind an extension to a loader")
-		.string("module-bind-post").describe("module-bind-post")
-		.string("module-bind-pre").describe("module-bind-pre")
-		.string("output-path").describe("output-path", "The output path for compilation assets")
-		.string("output-filename").describe("output-filename", "The output filename of the bundle")
-		.string("output-chunk-filename").describe("output-chunk-filename", "The output filename for additional chunks")
-		.string("output-source-map-filename").describe("output-source-map-filename", "The output filename for the SourceMap")
-		.string("output-public-path").describe("output-public-path", "The public path for the assets")
-		.string("output-jsonp-function").describe("output-jsonp-function", "The name of the jsonp function used for chunk loading")
-		.boolean("output-pathinfo").describe("output-pathinfo", "Include a comment with the request for every dependency")
-		.string("output-library").describe("output-library", "Expose the exports of the entry point as library")
-		.string("output-library-target").describe("output-library-target", "The type for exposing the exports of the entry point as library")
-		.string("records-input-path").describe("records-input-path", "Path to the records file (reading)")
-		.string("records-output-path").describe("records-output-path", "Path to the records file (writing)")
-		.string("records-path").describe("records-path", "Path to the records file")
-		.string("define").describe("define", "Define any free var in the bundle")
-		.string("target").describe("target", "The targeted execution environment")
-		.boolean("cache").describe("cache", "Enable in memory caching").default("cache", true)
-		.boolean("watch").alias("watch", "w").describe("watch", "Watch the filesystem for changes")
-		.boolean("watch-stdin").alias("watch-stdin", "stdin").describe("Exit the process when stdin is closed")
+		.boolean("help")
```

Comment:
Honestly, I prefer the old in this scenario. May be possible to resolve with a change in api.